zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/encoding/AuthInfo.java

```diff
@@ -73,25 +97,19 @@ public DirectBuffer toDirectBuffer() {
   }
 
   public Map<String, Object> toDecodedMap() {
-    switch (getFormat()) {
-      case JWT -> {
-        final String jwtToken = getAuthData();
-        return Authorization.jwtDecoder(jwtToken)
-            .withClaim(Authorization.AUTHORIZED_TENANTS)
-            .decode();
-      }
-      default -> {
-        return Map.of(
-            Authorization.AUTHORIZED_TENANTS, List.of(TenantOwned.DEFAULT_TENANT_IDENTIFIER));
-      }
+    if (claimsProp.hasValue()) {
+      return getClaims();
+    }
+    if (JWT.equals(getFormat())) {
+      final String token = getAuthData();
+      return new JwtDecoder(token).decode().getClaims();
     }
+    return Map.of();
   }
 
   @Override
   public String toString() {
-    String data = getAuthData();
-    data = data.isEmpty() ? "" : "." + data;
-    return formatProp.getValue().toString() + data;
+    return "AuthInfo{" + "authData=" + BufferUtil.bufferAsString(claimsProp.getValue()) + '}';
```

Comment:
:x: I don't think a `DocumentValue` is something we can/should interpret as a `String`. I'd suggest to just use the decoded map.

:wrench: The property name for the claims should be `claims`, not `authData`.

:wrench: I'd suggest to keep the old properties (format and authData) here, juuuust in case this goes wrong at some point.